appmap/_implementation/configuration.py

```diff
@@ -79,15 +128,18 @@ def function_in_set(fn, which):
         # fn isn't in a class, ignore it.
         return False
 
-    class_name += '.'
-    logger.debug(('class_name %s'
+    ret = (class_in_set(class_name, which)
+           or name_in_set(f'{class_name}.{fn_name}', which))
```

Comment:
You're right, it won't. Changing it would make it consistent with Ruby, sort of, and not consistent with Java, which uses `.` for instance and static methods. 

Also, `appmap-ruby` doesn't allow you to references to methods by name in `appmap.yml`. So it's inconsistent from that perspective as well.

Reply:
Being more Python than Ruby, to me personally, # syntax initially seemed pretty foreign. It is very unlikely for the `appmap.yml` to be shared between a Ruby and Python project, so I, personally, don't see the need for making it consistent across technologies.